chapter8.adoc

```diff
@@ -0,0 +1,30 @@
+[[chapter8]]
+[[Smsdedbg]]
+== `Smsdedbg`: Supervisor Domain External Debug
+
+[caption="Figure {counter:image}: ", reftext="Figure {image}"]
+[title= "External Debug for Supervisor Domain", id=Smsdedbg_img]
+image::Smsdedbg.png[]
+
+The <<MSDCFG>> CSR contains the `sdedbgen` bit that controls whether the
+current scheduled SD is external-debug enabled. This bit is context switched
+(along with rest of the `msdcfg`) per SD. The `medbgen` is an enable control
```

Comment:
The "enabled" terminology could be avoided as the DSM cannot determine whether external debug is enabled. It may be better to use "whether external debug is allowed for the currently scheduled SD".

Reply:
updated